CHANGES.rst

```diff
@@ -15,6 +14,8 @@ Other
 
 - Add wavelength tables for NIRSpec Drizzle cubepars reference file model. [#162]
 
+- Reduce interpolation vector length in NIRCam backwards transform
```

Comment:
This should go under a new 1.6 heading since 1.5 was released.

Reply:
Fixed - though I put it under 1.5.1 for now. Sorry, used to the new header being present post-build.